dd_game.yml

```diff
@@ -0,0 +1,184 @@
+  - freetype=2.11.0=h70c0345_0
+  - giflib=5.2.1=h7b6447c_0
+  - glib=2.69.1=h5202010_0
+  - greenlet=1.1.0=py38h709712a_0
+  - gst-plugins-base=1.14.0=h8213a91_2
+  - gstreamer=1.14.0=h28cd5cc_2
+  - icu=67.1=he1b5a44_0
+  - importlib-metadata=4.8.1=py38h06a4308_0
+  - iniconfig=1.1.1=pyhd3eb1b0_0
+  - intel-openmp=2021.3.0=h06a4308_3350
+  - jedi=0.17.0=py38_0
+  - jpeg=9d=h7f8727e_0
+  - kiwisolver=1.3.1=py38h2531618_0
+  - lcms2=2.12=h3be6417_0
+  - ld_impl_linux-64=2.35.1=h7274673_9
+  - libffi=3.3=he6710b0_2
+  - libgcc-ng=9.3.0=h5101ec6_17
+  - libgomp=9.3.0=h5101ec6_17
+  - libiconv=1.16=h516909a_0
+  - libpng=1.6.37=hbc83047_0
+  - libstdcxx-ng=9.3.0=hd4cf53a_17
+  - libtiff=4.2.0=h85742a9_0
+  - libuuid=1.0.3=h7f8727e_2
+  - libwebp=1.2.0=h89dd481_0
+  - libwebp-base=1.2.0=h27cfd23_0
+  - libxcb=1.14=h7b6447c_0
+  - libxml2=2.9.10=h68273f3_2
+  - lz4-c=1.9.3=h295c915_1
+  - matplotlib=3.3.2=0
+  - matplotlib-base=3.3.2=py38h817c723_0
+  - mccabe=0.6.1=py38_1
+  - mkl=2021.3.0=h06a4308_520
+  - mkl-service=2.4.0=py38h7f8727e_0
+  - mkl_fft=1.3.1=py38hd3c417c_0
+  - mkl_random=1.2.2=py38h51133e4_0
+  - more-itertools=8.10.0=pyhd3eb1b0_0
+  - ncurses=6.2=he6710b0_1
+  - numpy-base=1.21.2=py38h79a1101_0
+  - olefile=0.46=pyhd3eb1b0_0
+  - openssl=1.1.1l=h7f8727e_0
+  - packaging=21.0=pyhd3eb1b0_0
+  - pandas=1.1.3=py38he6710b0_0
+  - parso=0.8.2=pyhd3eb1b0_0
+  - pcre=8.45=h295c915_0
+  - pexpect=4.8.0=pyhd3eb1b0_3
+  - pickleshare=0.7.5=pyhd3eb1b0_1003
+  - pip=21.2.4=py38h06a4308_0
+  - pixman=0.38.0=h516909a_1003
+  - pluggy=0.13.1=py38h06a4308_0
+  - prompt-toolkit=3.0.20=pyhd3eb1b0_0
+  - ptyprocess=0.7.0=pyhd3eb1b0_2
+  - py=1.10.0=pyhd3eb1b0_0
+  - pycairo=1.20.1=py38hf61ee4a_0
+  - pycodestyle=2.7.0=pyhd3eb1b0_0
+  - pyflakes=2.3.1=pyhd3eb1b0_0
+  - pygments=2.10.0=pyhd3eb1b0_0
+  - pyparsing=2.4.7=pyhd3eb1b0_0
+  - pytest=6.1.2=py38h06a4308_0
+  - python=3.8.5=h7579374_1
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python_abi=3.8=2_cp38
+  - pytz=2021.3=pyhd3eb1b0_0
+  - rdkit=2020.09.3=py38h3f7302f_0
+  - readline=8.1=h27cfd23_0
+  - reportlab=3.5.68=py38hadf75a6_0
+  - setuptools=58.0.4=py38h06a4308_0
+  - sip=4.19.13=py38he6710b0_0
+  - six=1.16.0=pyhd3eb1b0_0
+  - sqlalchemy=1.4.22=py38h497a2fe_0
+  - sqlite=3.36.0=hc218d9a_0
+  - tk=8.6.11=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tornado=6.0.4=py38h1e0a361_2
+  - traitlets=5.1.0=pyhd3eb1b0_0
+  - wcwidth=0.2.5=pyhd3eb1b0_0
+  - wheel=0.37.0=pyhd3eb1b0_1
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.0.10=h7f98852_0
+  - xorg-libsm=1.2.2=h470a237_5
+  - xorg-libx11=1.7.2=h7f98852_0
+  - xorg-libxext=1.3.4=h7f98852_1
+  - xorg-libxrender=0.9.10=h7f98852_1003
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xz=5.2.5=h7b6447c_0
+  - zipp=3.6.0=pyhd3eb1b0_0
+  - zlib=1.2.11=h7b6447c_3
+  - zstd=1.4.9=haebb681_0
+  - pip:
+    - altair==4.1.0
+    - arcade==2.5.6
```

Comment:
Are we using all these (e.g. arcade)?

Reply:
Nope, we can deffo do a clean up of the installation file